TrackingProduction/Fun4All_FieldOnAllTrackersCalos.C

```diff
@@ -0,0 +1,411 @@
+/*
+ * This macro shows a minimum working example of running the tracking
+ * hit unpackers with some basic seeding algorithms to try to put together
+ * tracks, calorimeter cluster reconstruction, and track projected to claorimeter surface. There are some analysis modules run at the end which package
+ * hits, clusters, and clusters on tracks, calo clusters, and towers in calo clusters into trees for analysis.
+ * also provide the option to do full tracking detectors or TPC only, and drift velocity correction and EMcal radius correction
+ */
+
+#include <G4_ActsGeom.C>
+#include <G4_Magnet.C>
+#include <GlobalVariables.C>
+#include <Trkr_Clustering.C>
+#include <Trkr_RecoInit.C>
+#include <Trkr_Reco.C>
+#include <G4_Global.C>
+
+#include <caloreco/RawClusterBuilderTopo.h>
+
+#include <ffamodules/CDBInterface.h>
+#include <fun4all/Fun4AllDstInputManager.h>
+#include <fun4all/Fun4AllDstOutputManager.h>
+#include <fun4all/Fun4AllInputManager.h>
+#include <fun4all/Fun4AllUtils.h>
+#include <fun4all/Fun4AllOutputManager.h>
+#include <fun4all/Fun4AllRunNodeInputManager.h>
+#include <fun4all/Fun4AllServer.h>
+
+#include <phool/recoConsts.h>
+
+#include <track_to_calo/TrackCaloMatch.h>
+#include <track_to_calo/TrackToCalo.h>
+#include <track_to_calo/CaloOnly.h>
```

Comment:
Are these private libraries? We should remove them so that anyone can run this out of the box